test/indexer/test_hca_indexer.py

```diff
@@ -397,6 +399,21 @@ def test_project_contact_extraction(self):
             self.assertEqual({'Human Cell Atlas wrangler', None},
                              contributor_values['project_role'])
 
+    def test_diseases_field(self):
+        """
+        Index a bundle with a specimen `diseases` value that is differs from its donor `diseases` value
+        and assert that only the specimen's `diseases` value is in the indexed document.
+        """
+        self._mock_index(self.disease_bundle)
+
+        es_results = self._get_es_results()
+        for index_results in es_results:
+            bundle = index_results['_source']['bundles'][0]
+            if 'contents' in bundle:
```

Comment:
This is brittle way to distinguish between aggregate and original documents. For inspiration, take a look at test methods that were recently added to this class. Some of those tests (at least the one I added recently) has an assertion for both the original and for the aggregate. I think you should do that here, too. It's very straight-forward.